### ISSUE TITLE
Add Flow Types to AccessibilityExample.js

### ISSUE THREAD

**Issue by kacieb** [facebook/react-native#31329]
## Description
[AccessibilityExample.js](https://github.com/facebook/react-native/blob/master/packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js) does not have proper Flow types. Without Flow typing enforced, it is easy for bugs to be introduced when making changes to this file. This issue details the work to enforce Flow typing in this file.

Steps:
1. Add the `@flow` annotation to AccessibilityExample.js
2. Run `flow status` and fix any Flow errors that appear.
    * You can check out Flow's getting started guide here: https://flow.org/en/docs/getting-started/
    * You can check out Flow's React-specific documentation here: https://flow.org/en/docs/react/components/
    * An example file that is already annotated with `@flow`: [ScrollViewExample.js](https://github.com/facebook/react-native/blob/master/packages/rn-tester/js/examples/ScrollView/ScrollViewExample.js)
3. **Bonus:** Update the file to use [`@flow strict-local`](https://flow.org/en/docs/strict/#toc-strict-local)

## Expected Results
AccessibilityExample.js is annotated with proper flow types.

**Comment by ananta** [comment]
Hi @kacieb , I can give this a try!

**Comment by kacieb** [comment]
> Hi @kacieb , I can give this a try!

Awesome! Thanks so much for working on this. Please feel free to comment here if you have any questions. Once you're done, you can submit a PR for this and tag me as a reviewer!

**Comment by kacieb** [comment]
Thank you so much for fixing this issue @ananta! 🎉
